### PR TITLE
add person detection model

### PR DESCRIPTION
**Pull request by tushargoyal22**

### CODE REVIEW

**Review by mohitpandeyji — CHANGES_REQUESTED** [review]
@tushargoyal22
 keep only that model in project folder which you are using.

**Comment by tushargoyal22** [comment]
sir , the models are with different precisions, we can use any of them

**Comment by tushargoyal22** [comment]
so what i can do is to pick one model with a particular precision

**Comment by mohitpandeyji** [comment]
> so what i can do is to pick one model with a particular precision

also i am not sure whether we should push the models to git or not @vahiwe @tushargoyal22 

it makes the repository size to big

**Comment by tushargoyal22** [comment]
> > so what i can do is to pick one model with a particular precision
> 
> also i am not sure whether we should push the models to git or not @vahiwe @tushargoyal22
> 
> i makes the repository size to big

thats just 1mb file so i dont think it makes major effect

**Review by mohitpandeyji — COMMENTED** [review]
https://github.com/mohitpandeyji/askai.ga/blob/master/pull_request_template.md

please  check this and Every New feature will be done in one single branch and will be reviewed accordingly.
@tushargoyal22 @vahiwe

**Comment by tushargoyal22** [comment]
all files updated ,now its good to go

File: openvino_api/app.py [hunk 1]
```diff
@@ -64,6 +64,16 @@ def create_output_image(model_type, image, output):
                             (2 * scaler, 100 * scaler), cv2.FONT_HERSHEY_COMPLEX_SMALL,
                             1 * scaler, (255, 255, 255), 1 * scaler)
         return image
+
+    elif model_type == "PEDESTRIAN":
+        # Get only text detections above 0.5 confidence, set to 255
+        bbs = output
+        for boundingBox in bbs:
+            print(boundingBox[0], boundingBox[1])
```
Comment (mohitpandeyji) [inline openvino_api/app.py]: remove print

File: openvino_api/handle_models.py [hunk 2]
```diff
@@ -52,6 +52,26 @@ def handle_car(output, input_shape):
     return color_pred, type_pred
 
 
+def handle_pedestrian(output, input_shape):
+    '''
+    Handles the output of Pedestrian, caluculate the bounding boxes
+    '''
+    bbs = [] # array of [[p1.x , p1.y], [p2.x, p2.y]
+    output = output['detection_out']
+    print("input shape", input_shape)
```
Comment (mohitpandeyji) [inline openvino_api/handle_models.py]: remove

**Review by mohitpandeyji — COMMENTED** [review]
@vahiwe please check and test if possible .

File: openvino_api/app.py (hunk 1 above)
Comment (tushargoyal22) [reply openvino_api/app.py]: done

File: openvino_api/handle_models.py (hunk 2 above)
Comment (tushargoyal22) [reply openvino_api/handle_models.py]: done

**Pull request merged**